src/components/table/Table/index.js

```diff
@@ -212,6 +212,28 @@ const TableComponent = props => {
         gotoPage(0);
     }, [resetPageToOne]);
 
+    useSkipEffectOnMount(() => {
+        const newSort = params[getParamsKey(paramsPrefix, 'order')];
+        if (getSort(sortBy) !== newSort) {
+            setSortBy(getOrderArray(newSort));
+        }
+    }, [params[getParamsKey(paramsPrefix, 'order')]]);
+
+    useSkipEffectOnMount(() => {
+        const newPageSize = params[getParamsKey(paramsPrefix, 'pageSize')];
+        if (newPageSize !== pageSize) {
+            setPageSize(newPageSize);
+        }
+    }, [params[getParamsKey(paramsPrefix, 'pageSize')]]);
+
+    useSkipEffectOnMount(() => {
+        const newPage =
+            parseInt(params[getParamsKey(paramsPrefix, 'page')], 10) - 1;
```

Comment:
I'm going to be suuuuper nitpicky on this one, but: if `newPage` is NaN, `goToPage(NaN)` will probably not do anything good. There should probably also be a check that prevents `newPage` to be  <1.